src/components/labelled-date-input/LabelledDateInput.css

```diff
@@ -0,0 +1,12 @@
+.rdtPicker {
+  right: 0;
+}
+
+.labelled-date-input__input {
```

Comment:
This is almost copy pasted from `labelled-input__input` class. It's not good, but in the scope of this PR I think it's okay to don't fix it. We can later extract Input component, which will incapsulate this styles. And just pass this component to DateTime. But for now let's leave it as is.